docs/development-and-testing.md

```diff
@@ -53,6 +53,97 @@ $ ./kubectl-directpv --kubeconfig <PATH-TO-KUBECONFIG-FILE> info
 $ ./kubectl-directpv --kubeconfig <PATH-TO-KUBECONFIG-FILE> drives list
 ```
 
+## Testing with minikube
+
+1. Setup LVs
+
+The following script will create 4 LVs backed up by 4 loopback devices
+
+```bash
+sudo truncate --size=1G /tmp/disk-{1..4}.img
+for disk in /tmp/disk-{1..4}.img; do sudo losetup --find $disk; done
+devices=( $(for disk in /tmp/disk-{1..4}.img; do sudo losetup --noheadings --output NAME --associated $disk; done) )
+sudo pvcreate "${devices[@]}"
+vgname="vg0"
+sudo vgcreate "$vgname" "${devices[@]}"
+for lvname in lv-{0..3}; do sudo lvcreate --name="$lvname" --size=800MiB "$vgname"; done
+```
+
+2. Start minikube
+
+```bash
+minikube start --driver=none
+```
+
+3. Install directpv
+
+Install the freshly built version
+
+```bash
+./kubectl-directpv install --image directpv:<NEW_BUILD_TAG> --org <QUAY_USERNAME> --registry quay.io
+```
+
+4. Check if the drives are showing up
+
+```bash
+./kubectl-directpv drives list
+```
+
+5. Format the drives
+
+```bash
+./kubectl-directpv drives format --all
+```
+
+6. Apply the minio.yaml file
+
+Download and apply a sample MinIO deployment file available [here](https://github.com/minio/directpv/blob/master/minio.yaml)
+
+```bash
+kubectl apply -f minio.yaml
+```
+
```

Comment:
`sudo udevadm control --reload-rules && sudo udevadm trigger # in case of MiniKube drive sync issue.`

this could also be added as a note for troubleshooting.

Reply:
will add this in troubleshooting.md